data_steward/analytics/cdr_ops/rt_cdr_qc/cdr_deid_base_qa_report1.py

```diff
@@ -39,29 +39,142 @@
 # df will have a summary in the end
 df = pd.DataFrame(columns=['query', 'result'])
 
-# # 1 Verify that if a person has multiple SELECTion(Hispanic + other race) in pre_deid_com_cdr, the output in deid_base_cdr observation table should result in two rows - one for Ethnicity AND one for race.
+# ## Step1
+# - Verify the following columns in the deid_cdr Observation table have been set to null:
+#   - value_as_string
+#   - value_source_value
+#
+# has been done in first sql for deid, can be skipped here
+
+# ## Query 1.0
+#
+# Even though this option does not overtly identify a participant as AI/AN, it suggests so.  Therefore, these responses are generalized to "Other" and later deduplicate the rows that were created as a result
+#
+# We need a check for each part: We update both the value_source_concept_id and value_as_concept_id. Therefore, it is essential to include a validation check to confirm the success of the update.
```

Comment:
@MichaelSchmidt82 the following is added in the query 1.1 but that description belongs in here.
```
This check verifies records were suppressed. The observation_concept_id's are:
# - `40766241`
# - `1585389`
# - `43528428`
```